runtime/android/core_internal/src/org/xwalk/core/internal/XWalkInternalResources.java

```diff
@@ -35,7 +36,7 @@ private static void doResetIds(Context context) {
                     String generatedInnerClassName = innerClazz.getName().replace(
                             resourceClass, GENERATED_RESOURCE_CLASS);
                     try {
-                        generatedInnerClazz = classLoader.loadClass(generatedInnerClassName);
+                        generatedInnerClazz = appClassLoader.loadClass(generatedInnerClassName);
```

Comment:
This code helps to get resources after deprecated XWalkApplication and XWalkActivity.getResources()